<?xml version="1.0" encoding="UTF-8"?>
<testExecutions version="1">
    <file path="/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts">
        <testCase name="ProposalOptionResult Should throw ValidationException because there is no related_id" duration="3">
            <failure message="NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist"><![CDATA[NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist
    at new Exception (/Users/juha/Work/particl/particl-market/src/core/api/Exception.ts:24:15)
    at new NotFoundException (/Users/juha/Work/particl/particl-market/src/api/exceptions/NotFoundException.ts:17:9)
    at ListingItemTemplateService.<anonymous> (/Users/juha/Work/particl/particl-market/src/api/services/ListingItemTemplateService.ts:61:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:93:62)
    at <anonymous>]]></failure>
        </testCase>
        <testCase name="ProposalOptionResult Should throw ValidationException because we want to create a empty proposal option result" duration="0">
            <failure message="NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist"><![CDATA[NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist
    at new Exception (/Users/juha/Work/particl/particl-market/src/core/api/Exception.ts:24:15)
    at new NotFoundException (/Users/juha/Work/particl/particl-market/src/api/exceptions/NotFoundException.ts:17:9)
    at ListingItemTemplateService.<anonymous> (/Users/juha/Work/particl/particl-market/src/api/services/ListingItemTemplateService.ts:61:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:93:62)
    at <anonymous>]]></failure>
        </testCase>
        <testCase name="ProposalOptionResult Should create a new ProposalOptionResult" duration="0">
            <failure message="NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist"><![CDATA[NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist
    at new Exception (/Users/juha/Work/particl/particl-market/src/core/api/Exception.ts:24:15)
    at new NotFoundException (/Users/juha/Work/particl/particl-market/src/api/exceptions/NotFoundException.ts:17:9)
    at ListingItemTemplateService.<anonymous> (/Users/juha/Work/particl/particl-market/src/api/services/ListingItemTemplateService.ts:61:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:93:62)
    at <anonymous>]]></failure>
            <failure message="TypeError: Cannot read property &apos;id&apos; of undefined"><![CDATA[TypeError: Cannot read property 'id' of undefined
    at Object.<anonymous> (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:107:42)
    at Generator.next (<anonymous>)
    at /Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:96:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:92:16)
    at Object.test (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:104:68)
    at Object.asyncFn (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/jasmine-async.js:68:30)
    at resolve (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/queueRunner.js:38:12)
    at new Promise (<anonymous>)
    at mapper (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/queueRunner.js:31:21)
    at Promise.resolve.then.el (/Users/juha/Work/particl/particl-market/node_modules/p-map/index.js:42:16)
    at <anonymous>]]></failure>
        </testCase>
        <testCase name="ProposalOptionResult Should list ProposalOptionResults with our new create one" duration="4">
            <failure message="NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist"><![CDATA[NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist
    at new Exception (/Users/juha/Work/particl/particl-market/src/core/api/Exception.ts:24:15)
    at new NotFoundException (/Users/juha/Work/particl/particl-market/src/api/exceptions/NotFoundException.ts:17:9)
    at ListingItemTemplateService.<anonymous> (/Users/juha/Work/particl/particl-market/src/api/services/ListingItemTemplateService.ts:61:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:93:62)
    at <anonymous>]]></failure>
            <failure message="Error: expect(received).toBe(expected)"><![CDATA[Error: expect(received).toBe(expected)

Expected value to be (using ===):
  3
Received:
  0
    at Object.<anonymous> (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:133:45)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:93:62)
    at <anonymous>]]></failure>
        </testCase>
        <testCase name="ProposalOptionResult Should return one ProposalOptionResult" duration="0">
            <failure message="NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist"><![CDATA[NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist
    at new Exception (/Users/juha/Work/particl/particl-market/src/core/api/Exception.ts:24:15)
    at new NotFoundException (/Users/juha/Work/particl/particl-market/src/api/exceptions/NotFoundException.ts:17:9)
    at ListingItemTemplateService.<anonymous> (/Users/juha/Work/particl/particl-market/src/api/services/ListingItemTemplateService.ts:61:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:93:62)
    at <anonymous>]]></failure>
            <failure message="TypeError: Cannot read property &apos;id&apos; of undefined"><![CDATA[TypeError: Cannot read property 'id' of undefined
    at Object.<anonymous> (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:138:113)
    at Generator.next (<anonymous>)
    at /Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:96:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:92:16)
    at Object.test (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:137:66)
    at Object.asyncFn (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/jasmine-async.js:68:30)
    at resolve (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/queueRunner.js:38:12)
    at new Promise (<anonymous>)
    at mapper (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/queueRunner.js:31:21)
    at Promise.resolve.then.el (/Users/juha/Work/particl/particl-market/node_modules/p-map/index.js:42:16)
    at <anonymous>]]></failure>
        </testCase>
        <testCase name="ProposalOptionResult Should update the ProposalOptionResult" duration="1">
            <failure message="NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist"><![CDATA[NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist
    at new Exception (/Users/juha/Work/particl/particl-market/src/core/api/Exception.ts:24:15)
    at new NotFoundException (/Users/juha/Work/particl/particl-market/src/api/exceptions/NotFoundException.ts:17:9)
    at ListingItemTemplateService.<anonymous> (/Users/juha/Work/particl/particl-market/src/api/services/ListingItemTemplateService.ts:61:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:93:62)
    at <anonymous>]]></failure>
            <failure message="TypeError: Cannot read property &apos;id&apos; of undefined"><![CDATA[TypeError: Cannot read property 'id' of undefined
    at Object.<anonymous> (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:150:112)
    at Generator.next (<anonymous>)
    at /Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:96:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:92:16)
    at Object.test (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:145:66)
    at Object.asyncFn (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/jasmine-async.js:68:30)
    at resolve (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/queueRunner.js:38:12)
    at new Promise (<anonymous>)
    at mapper (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/queueRunner.js:31:21)
    at Promise.resolve.then.el (/Users/juha/Work/particl/particl-market/node_modules/p-map/index.js:42:16)
    at <anonymous>]]></failure>
        </testCase>
        <testCase name="ProposalOptionResult Should delete the ProposalOptionResult" duration="0">
            <failure message="NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist"><![CDATA[NotFoundException: Entity with identifier pmDP34sPsQnHAFN2r3B6MLj1ePTknm2Gh1 does not exist
    at new Exception (/Users/juha/Work/particl/particl-market/src/core/api/Exception.ts:24:15)
    at new NotFoundException (/Users/juha/Work/particl/particl-market/src/api/exceptions/NotFoundException.ts:17:9)
    at ListingItemTemplateService.<anonymous> (/Users/juha/Work/particl/particl-market/src/api/services/ListingItemTemplateService.ts:61:23)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:93:62)
    at <anonymous>]]></failure>
            <failure message="TypeError: Cannot read property &apos;id&apos; of undefined"><![CDATA[TypeError: Cannot read property 'id' of undefined
    at Object.<anonymous> (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:161:79)
    at Generator.next (<anonymous>)
    at /Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:96:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/Users/juha/Work/particl/particl-market/node_modules/tslib/tslib.js:92:16)
    at Object.test (/Users/juha/Work/particl/particl-market/test/integration/ProposalOptionResult.test.ts:159:66)
    at Object.asyncFn (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/jasmine-async.js:68:30)
    at resolve (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/queueRunner.js:38:12)
    at new Promise (<anonymous>)
    at mapper (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/queueRunner.js:31:21)
    at Promise.resolve.then.el (/Users/juha/Work/particl/particl-market/node_modules/p-map/index.js:42:16)
    at <anonymous>]]></failure>
            <failure message="Error: expect.assertions(1)"><![CDATA[Error: expect.assertions(1)

Expected one assertion to be called but only received zero assertion calls.
    at addAssertionErrors (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/setup-jest-globals.js:68:21)
    at Spec.attr.resultCallback (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/setup-jest-globals.js:111:9)
    at complete (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/jasmine/Spec.js:108:10)
    at queueRunnerFactory.then (/Users/juha/Work/particl/particl-market/node_modules/jest-jasmine2/build/jasmine/Spec.js:104:14)
    at <anonymous>]]></failure>
        </testCase>
    </file>
</testExecutions>